controllers/pipelines/experiment_controller.go

```diff
@@ -52,15 +51,16 @@ func (r *ExperimentReconciler) Reconcile(ctx context.Context, req ctrl.Request)
 
 	desiredProvider := desiredProvider(experiment, r.Config)
 
-	provider := pipelinesv1.Provider{
-		TypeMeta: metav1.TypeMeta{},
-		ObjectMeta: metav1.ObjectMeta{
-			Name: desiredProvider,
-		},
-		Status: pipelinesv1.ProviderStatus{},
+	logger.V(3).Info("found desired provider", "resource", desiredProvider)
```

Comment:
Don't think this log line is necessary

Reply:
changed in [f14cca8](https://github.com/sky-uk/kfp-operator/pull/354/commits/f14cca83d6fe1a7cb015482b23effbdc540042ee)